### PR TITLE
Implementing dual transcription + translation features 

### LINKED ISSUE #9: URGENT ENHANCEMENT: Adding the transcription + translation dual feature back
Add back the ability for users to be able to generate a transcription + translation csv

### LINKED ISSUE #16: Enhancement: Add support for English Only whisper models
To take full advantage of the CLI's prompt of _"Is your audio file in English?"_, the CLI needs to be able to load the english versions of the whisper models whenever available, if the audio file is all in English. 

To find the eng models, [refer to this link. ](https://github.com/openai/whisper). See section "Available models and languages".

### PR DESCRIPTION
**Pull request by carmen-chau**
### PR fix for issues: #9, #16

**Status: Completed rough implementation of #9, #16**

_More details regarding #9:_ Implemented a new method called `writing_comb_res_to_csv` that takes the lists from transcription and translation results. It creates a new list that combines the transcription and translation results. 

Note: If the amount of content from transcription and translation is NOT THE SAME, then for some fields, you would see the string "N/A" displayed. 

_More details regarding #16_ : Only the small and medium have pure .en models. Therefore, checks are only added for these 2 cases. There is only 1 large-v2 model.

### CODE REVIEW

File: backend/whisper_with_diarization_as_methods.py [hunk 1]
```diff
@@ -309,120 +302,89 @@ def write_list_to_csv(list_of_csv_content: List[str], output_csv_path: str, outp
...
-    #    output_csv_headers = ["Timestamps", "Speaker No", "Text[Eng]"]
-    #else:
-    #    output_csv_headers = ["Timestamps", "Speaker No", "Text[Orig Lang]", "Text[Eng]"]
-    the_date_time = str(datetime.datetime.now()).replace("-", "_").replace(" ", "_").replace(":", "_").replace(".", "_")
-    output_csv_path = destination_selection + "/" + the_date_time + "_" + "streetwhisperapp.csv"
-    translate_to_english = to_english_selection # True denotes that if audio file is not in english, you want to translate text to english. If False, text would be transcribed based on autodetected language from Whisper
+        output_format = "translation"
+    else: # reaching here means: process_selected == 'Transcription + Translation Only'
+        output_csv_headers = ["Timestamps", "Speaker No", "Text[Orig Lang]", "Text[Eng]"]
+        output_format = "transcribe&translate"
```
Comment (moonsdust) [inline backend/whisper_with_diarization_as_methods.py]: Not too sure about having & in the output file name.

**Review by moonsdust — APPROVED** [review]
This looks good! Feel free to merge it into main.

File: backend/whisper_with_diarization_as_methods.py (hunk 1 above)
Comment (carmen-chau) [reply backend/whisper_with_diarization_as_methods.py]: What would you suggest as the string for `output_format`? I think `transcribe_and_translate` would be too long.

Comment (moonsdust) [reply backend/whisper_with_diarization_as_methods.py]: Maybe `transcribe_translate`?

**Pull request merged**